src/executables/util_cs.c

```diff
@@ -3192,13 +3198,35 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
   if (master_node_name != NULL)
     {
       check_master_info = true;
+      ha_node_list_p = prm_get_string_value (PRM_ID_HA_NODE_LIST);
+      start_node = (char *) strchr (ha_node_list_p, '@');
+      if (start_node == NULL || ha_node_list_p == start_node)
+	{
+	  return NULL;
```

Comment:
```suggestion
	  goto print_applyinfo_usage
```

or add some error message

Reply:
In my opinion, instead of `goto print_applyinfo_usage`, `return EXIT_FAILURE;` looks so good because this is derived from wrong syntax.

Briefly,
AS-IS
**return NULL;**
TO-BE
**return EXIT_FAILURE;**